packages/babel-plugin-transform-es2015-modules-commonjs/src/index.js

```diff
@@ -129,6 +129,10 @@ export default function () {
   return {
     inherits: require("babel-plugin-transform-strict-mode"),
 
+    manipulateOptions(opts, parserOpts) {
+      parserOpts.plugins.push("objectRestSpread");
+    },
```

Comment:
@xtuc Yes! Unfortunately, I didn't found multiple inheritance too. Can you please point me if there util for merging configs?